components/dot_menu/dot_menu.tsx

```diff
@@ -399,6 +399,22 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
         });
     }
 
+    shortcutKey = (key: string) => {
+        return (
+            <span
+                className={classNames([
+                    'MenuItem__opacity',
+                    'MenuItem__right-decorator',
+                    {
+                        'MenuItem__text-color': !true,
+                    },
+                ])}
+            >
+                {key}
+            </span>
+        );
+    };
```

Comment:
OK, I will butt in here, since this could be done better.

First of all whenever you have a function that returns a component it is the best practice to name it `renderSomething`.

Then we have this: `'MenuItem__text-color': !true,`. That className will never find its way into the component with this condition, which makes the whole `classNames` call obsolete.

This is what I would do. Make a small memoized subcomponent out of it. That way React can best handle it.

This goes outside of the class component.

```suggestion
    type ShortcutKeyProps = {
        key: string;
    };

    const ShortcutKey = memo(({key}: ShortcutkeyProps) => !key ? null : (
        <span className={'MenuItem__opacity MenuItem__right-decorator'}>
            {key}
        </span>
    ));
```

Reply:
I reveresed the ternary condition to avoid the negated eslint check.

Reply:
>First of all whenever you have a function that returns a component it is the best practice to name it renderSomething.

Do we name this `renderShortcutKey`? :)

Reply:
this can now be removed with the `ShortcutKey` component